jobs/infra/fixtures/cleanup-env.sh

```diff
@@ -35,7 +35,7 @@ docker image prune -a --filter until=24h --force
 docker container prune --filter until=24h --force
 rm -rf /var/lib/jenkins/venvs
 rm -rf /var/lib/jenkins/.tox
-tmpreaper 5h /tmp
+sudo tmpreaper 5h /tmp
```

Comment:
"just black" you say?  must be that new AI black catching formatting errors like this :)